packages/driver/src/cy/commands/window.coffee

```diff
@@ -159,7 +159,7 @@ module.exports = (Commands, Cypress, cy, state, config) ->
 
       widthAndHeightAreWithinBounds = (width, height) ->
         _.every [width, height], (val) ->
-          val >= 20 and val <= 8000
+          val >= 0
```

Comment:
This won't catch `Infinity` or `NaN`, maybe this:

```suggestion
          _.isFinite(val) && val >= 0
```

Reply:
This is already checked with `widthAndHeightAreValidNumbers` (line 158, 201).

Reply:
Ah, I missed that, thanks